code/mob/living/carbon/human.dm

```diff
@@ -1390,6 +1390,8 @@
 			rendered += "[src.wear_id:registered]</span>"
 		else
 			rendered += "Unknown</span>"
+	else if (src.vdisfigured)
```

Comment:
better to change the logic of the if above than add another clause that does the same thing

Reply:
Cleaned this up.